### ISSUE TITLE
update to blackfriday v2.0.0 API

### ISSUE THREAD

**Issue by tamird** [cpuguy83/go-md2man#28]
https://github.com/russross/blackfriday/releases/tag/v2.0.0

**Comment by zevdg** [comment]
As a temporary measure, the import could at least be changed to gopkg.in/russross/blackfriday.v1

That or if this project were updated to use dep, a version constraint could be put on backfriday 

```toml
[[constraint]]
  name = "github.com/russross/blackfriday"
  version = "1.4"
```

**Comment by cpuguy83** [comment]
@zevdg It's already vendored.

**Comment by tamird** [comment]
Some tools flatten dependencies' vendor directories, fwiw.

**Comment by cpuguy83** [comment]
Happy to use dep if someone wants to convert it over.
Also if gopkg is the canonical import path, happy to use that as well.

**Comment by cpuguy83** [comment]
Being implemented in #33
